object/id/id.go

```diff
@@ -11,25 +11,31 @@ import (
 )
 
 // ID represents v2-compatible object identifier.
```

Comment:
```suggestion
// ID represents NeoFS object identifier.
```

Reply:
why? thought all public types should have such info